modules/refine-workflow/src/main/java/org/openrefine/exporters/StreamExporter.java

```diff
@@ -35,15 +35,15 @@ SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
 
 import java.io.IOException;
 import java.io.OutputStream;
-import java.util.Properties;
+import java.util.Map;
 
 import org.openrefine.ProjectMetadata;
 import org.openrefine.browsing.Engine;
 import org.openrefine.model.Grid;
 
 public interface StreamExporter extends Exporter {
 
-    public void export(Grid grid, ProjectMetadata projectMetadata, long projectId, Properties options, Engine engine,
+    public void export(Grid grid, ProjectMetadata projectMetadata, long projectId, Map<String, String> options, Engine engine,
```

Comment:
Compatibility can be preserved here by retaining the original method and adding the new method, both with default implementations.